backend/dataall/cdkproxy/stacks/dataset.py

```diff
@@ -249,19 +251,17 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
                 iam.PolicyStatement(
                     sid="GlueAccessCrawler",
                     actions=[
-                        "glue:GetDatabase",
-                        "glue:GetTableVersion",
+                        "glue:Get*",
```

Comment:
I see that we replaced all get actions with a wildcard get* and I wonder if we really need all new permissions? I'm asking because get* allows us to use many new methods and can be misused + violates least privilege principle 
This question is not only for glue, but also for other service policies in this PR like codecommit